### PR TITLE
Add arm32 build per pr.

### PR DESCRIPTION
**Pull request by jashook**
@mmitche PTAL

### CODE REVIEW

**Comment by jashook** [comment]
@dotnet-bot test ubuntu arm

**Comment by jashook** [comment]
@dotnet-bot test ubuntu arm64 build

**Comment by jashook** [comment]
@dotnet-bot test ci please

**Comment by mmitche** [comment]
It worked.

File: netci.groovy [hunk 1]
```diff
@@ -725,8 +726,15 @@ def static addTriggers(def job, def branch, def isPR, def architecture, def os,
             assert scenario == 'default'
             switch (os) {
                 case 'Ubuntu':
-                    Utilities.addGithubPRTriggerForBranch(job, branch, "${os} ${architecture} Cross ${configuration} Build", "(?i).*test\\W+${os}\\W+${architecture}.*")
-                    break
+                    switch(architecture) {
+                        case "arm":
+                            // Removing the regex will cause this to run on each PR.
+                            Utilities.addGithubPRTriggerForBranch(job, branch, "${os} ${architecture} Cross ${configuration} Build")
```
Comment (mmitche) [inline netci.groovy]: Do you want to turn this on by default if it fails today?

Comment (jashook) [inline netci.groovy]: Something changed from this morning. I have a clean build as of 5 minutes ago on the vm and hardware.

Comment (jashook) [inline netci.groovy]: Started the arm32 build to make sure the lab isn't doing something i'm not.

Comment (jashook) [inline netci.groovy]: Ya looks like its passing.

**Comment by jashook** [comment]
@dotnet-bot test ubuntu arm build

**Comment by manu-silicon** [comment]
:+1:

**Comment by jashook** [comment]
@dotnet-bot test ubuntu arm build

**Comment by jashook** [comment]
@dotnet-bot test ubuntu arm build

**Comment by jashook** [comment]
@dotnet-bot test ci

**Comment by jashook** [comment]
@dotnet-bot test ci please

**Pull request merged**